test/fixtures/gobierto_core/site_templates.yml

```diff
@@ -0,0 +1,4 @@
+madrid_participation_index:
```

Comment:
Could we add a test to check that this template is rendered instead of the regular view?